plugins/azure_ad_admin/icon_azure_ad_admin/actions/list_group_members/action.py

```diff
@@ -0,0 +1,63 @@
+import insightconnect_plugin_runtime
+from .schema import ListGroupMembersInput, ListGroupMembersOutput, Input, Output, Component
+
+# Custom imports below
+import requests
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+
+class ListGroupMembers(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="list_group_members",
+            description=Component.DESCRIPTION,
+            input=ListGroupMembersInput(),
+            output=ListGroupMembersOutput(),
+        )
+
+    def run(self, params={}):
+        # START INPUT BINDING - DO NOT REMOVE - ANY INPUTS BELOW WILL UPDATE WITH YOUR PLUGIN SPEC AFTER REGENERATION
+        group_id = params.get(Input.GROUP_ID)
+        # END INPUT BINDING - DO NOT REMOVE
+        url = f"https://graph.microsoft.com/v1.0/groups/{group_id}/members?$count=true"
```

Comment:
Can you move this to the `util/constants.py` file? I notice a few other endpoints in there so it would be nice to keep them all together
```
GRAPH_MICROSOFT_URL = "https://graph.microsoft.com/v1.0/{}/"


class Endpoint:
    USER_ID = GRAPH_MICROSOFT_URL + "users/{}"
    DEVICE_ID = GRAPH_MICROSOFT_URL + "devices/{device_id}"
    DEVICES = GRAPH_MICROSOFT_URL + "devices"
```

Reply:
Changes implemented